src/main/java/be/seeseemelk/mockbukkit/entity/OfflinePlayerMock.java

```diff
@@ -73,15 +74,21 @@ public boolean isOnline()
 	@Override
 	public boolean isOp()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return this.operator;
```

Comment:
This should get the status from the PlayerList, otherwise they can become desynced.